WinImageBuilder.psm1

```diff
@@ -543,6 +543,20 @@ function Add-DriversToImage {
     }
 }
 
+function Add-PackageToImage {
+    Param(
+        [Parameter(Mandatory=$true)]
+        [string]$winImagePath,
+        [Parameter(Mandatory=$true)]
+        [string]$packagePath
+    )
+    Write-Output ('Adding packages from "{0}" to image "{1}"' -f $packagePath, $winImagePath)

```

Comment:
use Write-Host (Write-output returns the string outputed to the function result)